<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientsInfo</class>
 <widget class="QWidget" name="PatientsInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QPushButton" name="prijemPacijentaButton">
     <property name="text">
      <string>Reception</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="refreshButton">
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="f_nameLabel">
         <property name="text">
          <string>Name: </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="nameEdit"/>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QFormLayout" name="formLayout_5">
       <item row="0" column="1">
        <widget class="QLineEdit" name="s_nameEdit"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="prezimeLabel">
         <property name="text">
          <string>Last name: </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="doctorLabel">
       <property name="text">
        <string>Doctor:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="doctorEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="filterButton">
       <property name="text">
        <string>Filter</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="deleteButton">
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="patientsView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
